### ISSUE TITLE
externals/soundtouch missing

### ISSUE THREAD

**Issue by robbilky** [PabloMK7/citra#32]
### Issue

externals/soundtouch directs to [https://github.com/PabloMK7/citra/blob/master/externals/soundtouch](url) I believe it is trying to get to this link [https://github.com/citra-emu/ext-soundtouch.git](url) which is obviously now dead. I didn't see a copy of ext-soundtouch in your repos like ext-libressl.

**Comment by CasualPokePlayer** [comment]
soundtouch is https://codeberg.org/soundtouch/soundtouch.git, which github won't redirect to.

**Comment by robbilky** [comment]
> soundtouch is https://codeberg.org/soundtouch/soundtouch.git, which github won't redirect to (you can find links to all submodules in the .gitmodules file).

my bad, thank you.

**Comment by wings110** [comment]
use https://github.com/libretro/citra 
add this [submodule "soundtouch"]
path = externals/soundtouch
url = https://codeberg.org/soundtouch/soundtouch.git
show error
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0
fatal: remote error: upload-pack: not our ref 060181eaf273180d3a7e87349895bd0cb6ccbf4a
fatal: Fetched in submodule path 'externals/soundtouch', but it did not contain 060181eaf273180d3a7e87349895bd0cb6ccbf4a. Direct fetching of that commit failed.
fatal:
*********** FAILED COMMAND ***********
. "${get_handler}"

*********** FAILED COMMAND ***********
${SCRIPTS}/get "${PKG_NAME}"

*********** FAILED COMMAND ***********
${SCRIPTS}/unpack "${PKG_NAME}" "${PARENT_PKG}"

I can't figure out where 060181eaf273180d3a7e87349895bd0cb6ccbf4a comes from. I cannot find any reference to it anywhere, but I might not know where to look. Any assistance?

**Comment by robbilky** [comment]
I opened issues on my fork of libretro-citra so we can discuss it there as I think this issue is out of the control of PabloMK7's version of Citra.

I can get mine to build so far with updated/downgraded externals, but had issues with dynarmic giving an error, because of the location of `a32.h`. I don't really know enough to fix this and I think it may just be incompatible with libretro-citra.

For soundtouch I linked to the codeberg repo ignored the commit and just changed the soundtouch paths in `Makefile.common`

Feel free to discuss here: [https://github.com/robbilky/citra/issues](url)
